### ISSUE TITLE
If the owner isn't going to fix this, can someone make a fork?

### ISSUE THREAD

**Issue by RichardMcKee** [kcxx/RetroWrapper#18]
(and post a link as an issue so other people notice and spread the word that there's a maintained fork with fixes?)

**Comment by NeRdTheNed** [comment]
Hey, if you still want a fork, I'm working on one! It's a bit on and off at the moment but I'll try to resolve some issues.

**Comment by RichardMcKee** [comment]
Ok! Thanks!

**Comment by JubeanHamukea** [comment]
> Hey, if you still want a fork, I'm working on one! It's a bit on and off at the moment but I'll try to resolve some issues.

Hi, is this still in development? I would love to see an up-to-date maintained fork of this! :)

**Comment by NeRdTheNed** [comment]
@JubeanHamukea Yes it is! The link is https://github.com/NeRdTheNed/RetroWrapper, I probably should've posted it here haha.
